packages/@storefront/core/src/core/utils.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="./../../custom_typings/gb-tracker-client.d.ts"/>
```

Comment:
Yeah that is required in order for the types to be incorporated in. Without that my IDE was saying that those particular types within the tracker service were "any" instead of actually taking me to the expected interface. With the change it everything works as expected.

Reply:
Found a work around, just had to include the custom_typings folder with the "includes" array in the tsconfig files within core.

Reply:
As per our in person discussion: I think that we should go with the triple slash directive where we reference the .d.ts file in order to make sure that both the unit tests and out ts files are pulling in the correct typings. Resolving this discussion.